launch_utils.py

```diff
@@ -12,7 +57,7 @@ def initialize():
 
 
 def create_instance():
-    return krpc.connect(name='Launch')
+    return krpc.connect(name='Launch', address=os.environ.get('KRPC_ADDRESS', '127.0.0.1')) # todo add this to all instances like this
```

Comment:
might need to use `host.docker.internal` in windows/WSL. I don't have a windows box to test that behavior out on, but I left some code for it in the docker compose